### PR TITLE
shortening the text

### PR DESCRIPTION
**Pull request by leftieFriele**
@trygve-lie @elisabethirgens @Andersos so I tried to shorten the text a little bit, I thought this page was a bit too long and some of the stuff could be left out. 
what u think?

### CODE REVIEW

File: views/about.hbs [hunk 1]
```diff
@@ -4,43 +4,27 @@
...
-</p>
 </article>
 
 <article>
     <h2>Code of Conduct</h2>
     <p>We want these couple of days to be truly enjoyable for everyone taking part in the conference.
-    Attendees, speakers, sponsors and volunteers must all abide by <a href="http://jsconf.com/codeofconduct.html">JSConf Code of Conduct</a>.
-    
-    </p>
+    Attendees, speakers, sponsors and volunteers must all abide by <a href="http://jsconf.com/codeofconduct.html">JSConf Code of Conduct</a>. Anyone reported not abide the CoC will be handled by the organizers.</p>
```
Comment (Andersos) [inline views/about.hbs]: I don't really like wording in "will be handled by the organizers"

File: views/about.hbs [hunk 2]
```diff
@@ -4,43 +4,27 @@
 
 <article>
   <h2>A non-profit, community driven conference</h2>
-<p>Web Rebels is for everyone who loves developing applications and services using web technology. We are proud members of <a href="http://jsconf.com/">the JSConf family of events</a>. Get a feel for what the conference is about, by checking out our <a href="https://vimeo.com/webrebels">videos</a> or <a href="http://flickr.com/webrebels">photos</a> from the previous three years.</p>
+<p>Web Rebels is for everyone who loves developing applications and services using web technology. We are proud members of <a href="http://jsconf.com/">the JSConf family of events</a>. We are a <a href="http://w2.brreg.no/enhet/sok/detalj.jsp?orgnr=997686659">registered not-for-profit organization</a> and is run by unpaid volunteers. If we make any profit from the conference we give 30% back to community groups throughout the Nordic region.</p>
```
Comment (Andersos) [inline views/about.hbs]: Changed "fun" by "run by unpaid"

File: views/about.hbs [hunk 3]
```diff
@@ -4,43 +4,27 @@
...
 
 <article>
-    <h2>Questions? Contact us!</h2>
-    <p> Any questions you might have, please do not hesitate to <a href="mailto:boyhowdy[ replace this bracket with a @ ]webrebels.org">shoot us an email</a> or ask <a href="http://twitter.com/web_rebels/">on Twitter</a>.</p>
-</article>
-<article>
-    <h2>Organizers</h2>
-    <p>Web Rebels is <a href="http://w2.brreg.no/enhet/sok/detalj.jsp?orgnr=997686659">registered as a non-profit organization</a> and is organized by unpaid volunteers.</p>
+<h2>16 invited speakers</h2>
+<p>For the <a href="./schedule">conference main track</a> we invite <a href="./speakers">16 speakers</a>. We do not have a CfP and our <a href="./policies">policy</a> is that speakers at the conference get paid for their travel and accommodation. We do not sell speaker slots and we do not accept sales pitches masked as presentations.
```
Comment (Andersos) [inline views/about.hbs]: Why CfP and not CFP?

**Comment by Andersos** [comment]
Made some comments. Looks good to be now.

File: views/about.hbs (hunk 3 above)
Comment (leftieFriele) [inline views/about.hbs]: spelling mistake :)

**Pull request merged**